lambdas/services/document_manifest_service.py

```diff
@@ -6,33 +6,56 @@
 from botocore.exceptions import ClientError
 from models.document_reference import DocumentReference
 from models.zip_trace import ZipTrace
+from pydantic import ValidationError
+from services.document_service import DocumentService
 from services.dynamo_service import DynamoDBService
 from services.s3_service import S3Service
 from utils.audit_logging_setup import LoggingService
-from utils.exceptions import ManifestDownloadException
+from utils.exceptions import (DocumentManifestServiceException,
+                              DynamoDbException)
 
 logger = LoggingService(__name__)
 
 
 class DocumentManifestService:
-    def __init__(
-        self,
-        nhs_number: str,
-        documents: list[DocumentReference],
-        zip_output_bucket: str,
-        zip_trace_table: str,
-    ):
+    def __init__(self, nhs_number):
         self.s3_service = S3Service()
         self.dynamo_service = DynamoDBService()
+        self.document_service = DocumentService()
+
         self.nhs_number = nhs_number
-        self.documents = documents
-        self.zip_output_bucket = zip_output_bucket
-        self.zip_trace_table = zip_trace_table
+        self.documents: list[DocumentReference] = []
         self.zip_file_name = f"patient-record-{self.nhs_number}.zip"
```

Comment:
As above, I think we should make this as it's needed and not in the init

Reply:
Similar here, it's passed to the s3 service in 2 separate methods so seems sensible to keep as an object variable, otherwise we'd just be duplicating the same string